02/app.js

```diff
@@ -1,7 +1,116 @@
 
 /* rozwiązanie z pętlą for */
-const x = 4;
 
 
+function getSum(numberFromUser, multiply, add) {
 
-/* rozwiązanie z pętlą while  */
\ No newline at end of file
+   const x = numberFromUser();
+
+   for(let i=1; i<=9; i++) {
+
+       const y = multiply(i , x);
+       const z = add(i , x);
+
+        const sum = i * x;
+
+        console.log('Your' , i , 'multiply resault is:' , sum , 'because' , y , sum , '(' , z , sum , ')');
+
+    }
+}
+
+
+getSum(getUserNumber, getMultiplySequence, getAddSequence);
+
+
+function getAddSequence(a,b) {
+
+    let addSequence = "";
+    i=1;
```

Comment:
Czy to ma być zmienna globalna? Raczej nie więc brakuje dekalracji tj. `let i = 1`;